packages/browser/src/client/popup.ts

```diff
@@ -0,0 +1,22 @@
+const showPopupWarning = (name: string) => (...params: any[]) => {
+  const formatedParams = params.map(p => JSON.stringify(p)).join(', ')
+
+  console.warn(`Vitest encountered a \`${name}\` call with parameters: ${
+      formatedParams}. Vitest cannot handle \`${
+      name}\` by default in the browser, so it returned null. If you need to handle this case, you can mock the \`${
+      name}\` call yourself like this:
+
+\`\`\`
+vi.spyOn(window, '${name}').mockReturnValue('your value')
```

Comment:
The text was just an example. We also need to import `vi` from `vitest` here to show where it comes from to the new users.

I also don't think using "`" does anything here for the preview. Do you have pictures how it looks like in browser and in the terminal? Can we improve the look somehow?

Reply:
![image](https://user-images.githubusercontent.com/37929992/227638876-d5336fdf-b145-432b-8874-ee080aa27908.png)

Reply:
Yes, I'll update it. but around the `, I think they separate the text from the code properly.

![image](https://user-images.githubusercontent.com/37929992/227640135-c3311c48-9cfa-41a7-95be-f3160f196178.png)

Reply:
Here's the new version, I also added some docs.

![image](https://user-images.githubusercontent.com/37929992/227653601-6260541f-69b6-45d9-9893-191f3246f917.png)

Reply:
![image](https://user-images.githubusercontent.com/37929992/227659298-0b9997e9-f5ec-49d3-b4ba-b99989340db1.png)